src/rpc/methods/f3.rs

```diff
@@ -567,6 +576,57 @@ impl RpcMethod<2> for SignMessage {
     }
 }
 
+pub enum GetManifestFromContract {}
+
+impl GetManifestFromContract {
+    pub fn create_eth_call_message(contract: EthAddress) -> EthCallMessage {
+        // method ID of activationInformation()
+        static METHOD_ID: Lazy<EthBytes> =
+            Lazy::new(|| EthBytes::from_str("0x2587660d").expect("Infallible"));
```

Comment:
We already have this elsewhere. It'd be great to make it a constant and document it, e.g., with a link to the FRC.

Reply:
Could you point to me where a similar method id lives? 
BTW, added a link to where the constant is defined

Reply:
Ah okay, I thought you're duplicating the shed one.